vm/interpreter/src/default_runtime.rs

```diff
@@ -101,6 +104,8 @@ where
         origin: Address,
         origin_nonce: u64,
         num_actors_created: u64,
+        pre_ignition: GenesisInfo,
+        post_ignition: GenesisInfo,
```

Comment:
This should not be attached to the runtime, this does not allow for swapable circ supply calculations